series/type-bool.go

```diff
@@ -19,15 +20,12 @@ func (e *boolElement) Set(value interface{}) {
 			e.nan = true
 			return
 		}
-		switch strings.ToLower(value.(string)) {
-		case "true", "t", "1":
-			e.e = true
-		case "false", "f", "0":
-			e.e = false
-		default:
+		v, err := strconv.ParseBool(value.(string))
```

Comment:
I agree with this change and will merge it here, but again, this should be a separate PR in the future